package.json

```diff
@@ -11,21 +11,24 @@
   ],
   "scripts": {
     "analyze": "size-limit --why",
-    "build": "dts build",
-    "lint": "dts lint --config \"./.prettierrc.yml\"",
-    "prepare": "dts build",
+    "build": "tsc",
+    "lint": "eslint --config \"./.eslintrc.json\" \"**/*.{json,ts,yml}\" && prettier --config \"./.prettierrc.json\" --write \"**/*.{json,md,js,ts,yml}\"",
```

Comment:
consider breaking this up into several commands

Reply:
separated lint and prettier 👍🏼